src/comps/ui/nav/NavBar.tsx

```diff
@@ -1,379 +1,530 @@
+                        onClick={() => {
+                            navigate("/");
+                            setTimeout(() => setIsHovered(false), 300);
+                        }}
+                    >
+                        Home
+                    </span>
+                    <i className="bx bx-group"></i>
+                    <span
+                        className={"transition-colors hover:text-gray-300"}
+                        style={{
+                            marginLeft: "3px",
+                            position: "relative",
+                            top: "2px",
+                            cursor: "pointer",
+                        }}
+                        onMouseEnter={() => setIsHovered(true)}
+                        onClick={() => {
+                            navigate("/catalog");
+                            setTimeout(() => setIsHovered(false), 300);
+                        }}
+                    >
+                        StuyActivities
+                    </span>
+                    <i className="bx bx-calendar"></i>
+                    <span
+                        className={"transition-colors hover:text-gray-300"}
+                        style={{
+                            marginLeft: "3px",
+                            position: "relative",
+                            top: "2px",
+                            cursor: "pointer",
+                        }}
+                        onMouseEnter={() => setIsHovered(true)}
+                        onClick={() => {
+                            navigate("/meetings");
+                            setTimeout(() => setIsHovered(false), 300);
+                        }}
+                    >
+                        Meetings
+                    </span>
+                    <i className="bx bx-note"></i>
+                    <span
+                        className={"transition-colors hover:text-gray-300"}
+                        style={{
+                            marginLeft: "3px",
+                            position: "relative",
+                            top: "2px",
+                            cursor: "pointer",
+                        }}
+                        onMouseEnter={() => setIsHovered(true)}
+                        onClick={() => {
+                            window.location.href = "https://vote.stuysu.org";
+                            setTimeout(() => setIsHovered(false), 300);
+                        }}
+                    >
+                        Voting Site
+                    </span>
+                    <i className="bx bx-glasses opacity-50"></i>
+                    <span
+                        style={{
+                            opacity: 0.5,
+                            marginLeft: "3px",
+                            position: "relative",
+                            top: "2px",
+                        }}
+                    >
+                        Opportunities
```

Comment:
Probably hide this for now since Opportunities will not be ready for quite some time (this is SU IT's Half Life 3)

Reply:
By valve standards our opportunities page will not be be ready forever

Reply:
:heavy_check_mark: